qemu/tests/nic_acpi_index_boot.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import re
+
+from virttest import utils_net
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    When "acpi-index=N" is enabled, NIC name should always be "ethN"
+
+    1) Boot up guest with a single nic, add nic option "acpi-index=1"
+    2) Remove "biosdevname=0" and "net.ifname=0" from kenrel command line
+    3) Reboot guest
+    4) Check the nic name, the guest nic name enoN == acpi-index=N
+    5) Boot up guest without nic
+    6) Add nic devices through monitor cmd and check if it added
+    7) Check the nic name, the guest nic name enoN == acpi-index=N
+    8) Check if new interface gets ip address
+    9) Ping guest new ip from host
+    10) Delete nic device, then hot plug nic with same slot and different
+    acpi-index number, repeat step 7~9
+    11) Hot plug the nic with different slot and same acpi-index number, repeat
+    step 7~9
```

Comment:
The document is not the same as your steps.

Reply:
Changed the document with this update.